tests/test_stac.py

```diff
@@ -3,6 +3,20 @@
 import pytest
 
 from stactools.ecmwf_forecast import stac
+import json
+import os
+import urllib.request
+
+blob_file = ("https://ai4edataeuwest.blob.core.windows.net/ecmwf/20231019/00z/"
+         "0p4-beta/wave/20231019000000-0h-wave-fc.grib2")
+
+local_files = ["20231019000000-0h-wave-fc.grib2",
+               "20231019/00z/0p4-beta/wave/20231019000000-0h-wave-fc.grib2"]
+for i in range(len(local_files)):
```

Comment:
```suggestion
for i, local_file in enumerage(local_files):
```

then use `local_file` below instead of `local_files[i]`.

Reply:
Done.